package/ci/travis-emscripten.sh

```diff
@@ -1,12 +1,19 @@
 #!/bin/bash
 set -ev
 
+# Get ninja-build
+wget https://github.com/ninja-build/ninja/releases/download/v1.8.2/ninja-linux.zip
+unzip ninja-linux.zip
+mv ninja /usr/local/bin/
+ninja --version
```

Comment:
Heh. I suppose `apt install ninja` just doesn't work on docker, right?

Reply:
Oh, actually sure, that would work, if there's a package (I assume it's called ninja-build, though?)

Reply:
In the yml it's listed among the packages, but I think it's ninja-build, yes.